compute/k8s-services/vars.tf

```diff
@@ -1256,6 +1256,18 @@ variable "grafana_agent_traces_enabled" {
   description = "Enable traces or not. Default: true"
 }
 
+variable "grafana_agent_resource_memory_limit" {
+  type = string
+  default = "20Gi"
+  description = "Set resource memory limits on Grafana Agent container"
+}
+
+variable "grafana_agent_resource_memory_request" {
+  type = string
+  default = "4Gi"
```

Comment:
I think we ought to set this to the same value as max, or at least closer to limit fragmentation.

Reply:
We can do that, but without a dedicated node group it might be difficult to find an available node with that high a request.